test/onigumo_cli_test.exs

```diff
@@ -35,6 +42,20 @@ defmodule OnigumoCLITest do
       assert Onigumo.CLI.main(["downloader"]) == tmp_dir
     end
 
+    @tag :tmp_dir
+    test("run CLI 'downloader' with '--working-dir' switch", %{tmp_dir: tmp_dir}) do
+      expect(OnigumoDownloaderMock, :main, fn root_path -> root_path end)
+
+      assert Onigumo.CLI.main(["downloader", "--working-dir", tmp_dir]) == tmp_dir
+    end
+
+    @tag :tmp_dir
+    test("run CLI 'downloader' with '-C' switch", %{tmp_dir: tmp_dir}) do
+      expect(OnigumoDownloaderMock, :main, fn root_path -> root_path end)
+
+      assert Onigumo.CLI.main(["downloader", "-C", tmp_dir]) == tmp_dir
+    end
```

Comment:
Could this be parametrized, using for with a list of valid arguments. Not only it’s the same test repeated twice, but also it’s the same switch, just with an alias.

```suggestion
    @working_dir_switches ["--working-dir", "-C"]

    @tag :tmp_dir
    for switch <- @working_dir_switches do
      test("run CLI 'downloader' with #{inspect(switch)} switch", %{tmp_dir: tmp_dir}) do
        expect(OnigumoDownloaderMock, :main, fn root_path -> root_path end)

        assert Onigumo.CLI.main(["downloader", switch, tmp_dir]) == tmp_dir
      end
```

Reply:
Absolutely, thank you.